asset/asset.go

```diff
@@ -18,6 +21,13 @@ import (
 	"github.com/lightningnetwork/lnd/tlv"
 )
 
+const (
+	// MaxAssetNameLength is the maximum byte length of an asset's name.
+	// This byte length is equivalent to character count for single-byte
+	// UTF-8 characters.
+	MaxAssetNameLength = 64
```

Comment:
If we want to enforce this as a protocol level check, then we also need to apply this during proof verification.

Reply:
Added a commit to address this.